src/ui/Snackbar/SnackbarList.tsx

```diff
@@ -0,0 +1,49 @@
+import { AnimatePresence, motion, Variants } from 'framer-motion';
+import { ComponentProps } from 'react';
+import Snackbar from './Snackbar';
+
+type TSnackbar = ComponentProps<typeof Snackbar> & {
+  id: string;
+};
+type Props = {
+  className?: string;
+  snacks: TSnackbar[];
+};
+function SnackbarList({ className, ...props }: Props) {
+  return (
+    <ol className={className} tw="flex flex-col gap-sm">
+      <AnimatePresence>
```

Comment:
Can you make the snackbar enter and exit from the right with a sliding effect? This's what Minimals does I think

Reply:
pushed this change right before you reviewed it.